### DISCUSSION TITLE
Ingest and store audio only stream

### DISCUSSION THREAD

**Discussion by ivan-carbon** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-js#320] [Q&A]
I wonder if it possible to send an audio-only stream without video data? The instructions in the code mention that "both audio and video must be sent for WebRTC Ingestion and Storage." I'm wondering if there is a way to support audio-only mode without mocking a video signal.

**Comment by stefankiesz** [comment]
Hi Ivan, at the moment audio-only is not supported, we will take this use case into account for future support expansion considerations.

**Reply by matt-carbon** [reply]
@stefankiesz  I'm not sure I understand how audio only isn't supported. Can you help clarify how something like this example works? https://github.com/schuettc/amazon-chime-sdk-call-analytics-reader

**Comment by angangwa** [comment]
Hi, is there any update on this? Even some sample code will help. How do we send just audio data without frames? This is blocking us from a POC and recommending usage of Kinesis.

**Reply by BenOmerDev** [reply]
Hi @angangwa, would you describe the industry and/or use case being explored for audio-only ingest via WebRTC. This request now has some attention in our product service team.

**Reply by BenOmerDev** [reply]
Feel free to email us with details at kinesis-video-support@amazon.com

**Reply by angangwa** [reply]
Apologies for the delay.

Here are the details I am able to share.

- Industry: FSI
- Use case: We are evaluating Amazon Connect and its feature to stream call audio to KVS. In order to build a production grade applicaiton, we will need the ability to mock audio streams. Moreover, we are interested in making the call centre audio available to multiple consumers from KVS - we anticipate this ability will be helpful beyond the Amazon Connect use-case. 
- Current workaround: Using audio files, evaluating custom application for streaming needs.
- Scale: Whilst I am unable to share specifics, our MVP will deal with a portion of calls amounting to ~5k calls / month with an average duration of 45 minutes. 

Hope this helps. I will also send out an email with these details. Appreciate your interest and thank you for responding back.
